AztecTests/TextViewTests.swift

```diff
@@ -38,6 +38,15 @@ class TextViewTests: XCTestCase {
         return richTextView
     }
 
+    let anotherFont = UIFont(name:"HelveticaNeue", size: 14)!
```

Comment:
"anotherFont" feels very generic and is not very clear for maintainability.

Can we add a comment explaining why this is used, plus probably rename this to something like `nonStandardSystemFont`?